acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/pubsub/NewPendingTransactionAcceptanceTest.java

```diff
@@ -70,7 +70,7 @@ public void transactionRemovedByChainReorganisationMustPublishEvent() throws Exc
 
     final Condition atLeastLighterForkBlockNumber = blockchain.blockNumberMustBeLatest(minerNode);
 
-    cluster.stop();
+    cluster.close();
```

Comment:
Why `close` instead of `stop` here?  I'd have expected `close` to only be called as part of tear down.

Reply:
This method will be removed in next PR ... I'll revert these changes so that it doesn't get reflect in this PR.